MSAL.podspec

```diff
@@ -20,34 +20,45 @@ Pod::Spec.new do |s|
     :tag => s.version.to_s,
     :submodules => true
   }
-
-  s.pod_target_xcconfig = { 'CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF' => 'NO', 'HEADER_SEARCH_PATHS' => "$SRCROOT/MSAL"}
+  
   s.default_subspecs ='app-lib'
   
   s.prefix_header_file = "MSAL/src/MSAL.pch"
   s.header_dir = "MSAL"
 
   s.subspec 'app-lib' do |app|
-    app.source_files = "MSAL/src/**/*.{h,m}", "MSAL/src/native_auth/**/*.{h,m,swift}", "MSAL/IdentityCore/IdentityCore/src/**/*.{h,m}", "MSAL/module.modulemap"
+    app.pod_target_xcconfig = { 'CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF' => 'NO' }
+    app.source_files = "MSAL/src/**/*.{h,m}", "MSAL/IdentityCore/IdentityCore/src/**/*.{h,m}", "MSAL/src/native_auth/public/*.{h,m}"
```

Comment:
Regarding the line: `..., "MSAL/src/native_auth/public/*.{h,m}"`, shouldn't we exclude everything native_auth related?

Reply:
I added the reason we added this line in the "additional information" in the PR description. In brief, we add the Objective-C code to avoid having two different MSAL.h files